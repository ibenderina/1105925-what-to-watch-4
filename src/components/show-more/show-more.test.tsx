import * as React from "react";
import * as renderer from "react-test-renderer";
import ShowMore from "@components/show-more/show-more";

it(`Should Show more render correctly`, () => {
  const tree = renderer.create(
    <ShowMore handleShowButtonClick={() => {}}/>
  );
  expect(tree.toJSON()).toMatchSnapshot();
});
