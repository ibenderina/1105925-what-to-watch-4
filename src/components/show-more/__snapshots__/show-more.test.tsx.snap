// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Show more render correctly 1`] = `
<div
  className="catalog__more"
>
  <button
    className="catalog__button"
    onClick={[Function]}
    type="button"
  >
    Show more
  </button>
</div>
`;
